nilearn/plotting/html_stat_map.py

```diff
@@ -1,148 +1,449 @@
-    rgb = cmap(x)[:, :3]
-    papaya['positive_cmap'] = _rgb_to_papaya(y, rgb)
-    return papaya
-
-
-def _rgb_to_papaya(x, cmap):
-    """Transform (values, rgb array) to format expected by papaya"""
-    cmap = [[float(m)] + list(map(float, col)) for m, col in zip(x, cmap)]
-    return cmap
-
-
-def view_stat_map(stat_map_img, threshold=None, bg_img='MNI152',
-                  vmax=None, cmap='cold_hot', symmetric_cmap=True):
+def view_stat_map(stat_map_img, bg_img='MNI152', cut_coords=None,
+                  colorbar=True, title=None, threshold=None, annotate=True,
+                  draw_cross=True, black_bg='auto', cmap=cm.cold_hot,
+                  symmetric_cbar='auto', dim='auto', vmax=None,
+                  resampling_interpolation='continuous', n_colors=256,
+                  opacity=1, **kwargs):
     """
-    Insert a surface plot of a surface map into an HTML page.
+    Intarctive viewer of a statistical map, with optional background
 
     Parameters
     ----------
     stat_map_img : Niimg-like object
         See http://nilearn.github.io/manipulating_images/input_output.html
-        The statistical map image. Should be 3D or
-        4D with exactly one time point (i.e. stat_map_img.shape[-1] = 1)
-
-    threshold : str, number or None, optional (default=None)
-        If None, no thresholding.
-        If it is a number only values of amplitude greater
-        than threshold will be shown.
-        If it is a string it must finish with a percent sign,
-        e.g. "25.3%", and only values of amplitude above the
-        given percentile will be shown.
-
-    bg_img : Niimg-like object, optional (default='MNI152')
+        The statistical map image.
+    bg_img : Niimg-like object (default='MNI152')
         See http://nilearn.github.io/manipulating_images/input_output.html
         The background image that the stat map will be plotted on top of.
         If nothing is specified, the MNI152 template will be used.
-
-    vmax : float, optional (default=None)
-        Upper bound for plotting
-
-    cmap : str or matplotlib colormap, optional
-
-    symmetric_cmap : bool, optional (default=True)
-        Make colormap symmetric (ranging from -vmax to vmax).
-        Set it to False if you are plotting an atlas or an anatomical image.
+        To turn off background image, just pass "bg_img=False".
+    cut_coords : None, a tuple of floats, or an integer (default None)
+        The MNI coordinates of the point where the cut is performed
+        as a 3-tuple: (x, y, z). If None is given, the cuts is calculated
+        automaticaly.
+        This parameter is not currently supported.
+    colorbar : boolean, optional (default True)
+        If True, display a colorbar next to the plots.
+    title : string or None (default=None)
+        The title displayed on the figure (or None: no title).
+        This parameter is not currently supported.
+    threshold : str, number or None  (default=None)
```

Comment:
this breaks if a string is given, because find_cut_coords is passed a string. please use check_threshold

Reply:
was also brought up by @kchawla-pi . Will work on that.

Reply:
mmhh, now I realize I did not understand the doc. I thought str referred to the case 'auto'. But what you are saying is that threshold="90%" should be supported?